packages/contentstack-import/package.json

```diff
@@ -34,7 +34,7 @@
     "@types/mocha": "^8.2.2",
     "@types/node": "^14.14.32",
     "@types/sinon": "^10.0.2",
-    "@types/tar": "^4.0.3",
+    "@types/tar": "^6.1.3",
```

Comment:
Is there any breaking changes for this package?

Reply:
No